pom.xml

```diff
@@ -38,7 +38,7 @@
     <mockwebserver.version>4.11.0</mockwebserver.version>
 
     <!-- plugins -->
-    <openapi-generator.version>6.2.1</openapi-generator.version>
+    <openapi-generator.version>7.4.0</openapi-generator.version>
```

Comment:
using newest (anything ≥ 7.5.0) breaks a test in `EntityTypeControllerTest`, assuming due to a change that occurred in initialization/default values.